### ISSUE TITLE
dependencies not referenced correctly

### ISSUE THREAD

**Issue by mmestas** [michaelbushe/cytoscape-angular#4]
I'm using this in an angular 12 application and got several reference errors from this library.  Eventually, I got it to work by changing the node-module directly by adding/changing the references in cytoscope-angular.js : 

`import { ButtonModule } from 'primeng/button';
import { DropdownModule } from 'primeng/dropdown';
import { FieldsetModule } from 'primeng/fieldset';
import { InputSwitchModule } from 'primeng/inputswitch';
import { InputTextModule } from 'primeng/inputtext';
import { OverlayPanelModule } from 'primeng/overlaypanel';
import { ProgressBarModule } from 'primeng/progressbar';
import { ProgressSpinnerModule } from 'primeng/progressspinner';
import { TooltipModule } from 'primeng/tooltip';
import { SpinnerModule } from 'primeng/spinner';
import { AutoCompleteModule } from 'primeng/autocomplete';`

and 

`import * as cytoscape from 'cytoscape';`

The errors were that cytoscape was not defined and the above primeng references had no exported module xx.  It seems that each module has to be referenced individually and that cytoscape cannot be globally referenced without declaring it in an import.

**Comment by michaelbushe** [comment]
Yes the latest version should have the new package names to catch up with primeng changes. 
I’m still having trouble with the CytoscapeAngularModule being recognized, I’ll see if your import helps with that.

**Comment by RobRoyce** [comment]
I'm getting a `cityscape is not defined` error and it seems to be related to the above. @mmestas would you be able to share what your imports look like in your Angular component?

I followed the example project to a T but still nothing 🤷‍♂️

**Comment by RobRoyce** [comment]
The odd part is I actually get a canvas with a single node (which is outlined by a "loading" animation, presumably). So cityscape is, in fact, imported into the project.
